src/qibolab/serialize.py

```diff
@@ -173,6 +180,13 @@ def dump_runcard(platform: Platform, path: Path):
         path (pathlib.Path): Path that the yaml file will be saved.
     """
 
+    kernels = Kernels()
+    for qubit in platform.qubits.values():
+        if qubit.kernel is not None:
+            kernels[qubit.name] = qubit.kernel
+            qubit.kernel = None
+    Kernels(kernels).dump(Path(__file__).parent / "dummy" / KERNELS_FILE)
```

Comment:
Two reasons, the first I forgot and the second it's related to this. https://github.com/qiboteam/qibolab/pull/758#discussion_r1454893459 I wanted to use the `platform.name` but the dummy has two, I think the best would be something like this ?

```python
name = platform.name
if platform.name == "dummy_couplers":
    name = "dummy"
Kernels(kernels).dump(Path(__file__).parent / name / KERNELS_FILE)
```

Reply:
For as long as we need to maintain two dummy platforms, couldn't we have actually two folders?

Reply:
I am not that much concerned about the dummy, the implemented solution should be fine. However, this is still a bit problematic because it is asymmetric to the loading. 

If I understand it correctly, when you create the platform you will load the kernels from whatever `extras_folder` is passed to `load_qubits`. This is typically located together with the platform runcard. Then when you (or qibocal) call `dump` it is using `Path(__file__).parent / name` to determine the directory, where `__file__` is the serialize.py. Therefore it will dump the new kernels inside your qibolab installation, which is in general a different place than where the runcards are storred. Is this what we want?

Reply:
That's the other solution, I'm fine with any.

Reply:
We want that for the dummy, but not for the other platforms. I will think of something then.

Reply:
I think `dump_runcard` is automatically called by qibocal after every routine. So if you add kernels to any platform (eg. iqm5q) and you run any qibocal routine, it will try to dump the kernels to a directory like
```
qibolab/src/qibolab/iqm5q/kernels.npz
```
and will probably fail if there is no folder `iqm5q` within qibolab. At least that's what I think, I didn't do the actual test.

Reply:
Tests are failing because of that, can I add the extras_folder path as a atribute for the platform ?

Reply:
That’s probably acceptable for now and we can improve once we implement the folder structure proposed by @alecandido.

Reply:
I agree we could go incrementally (eventually we can even use an env var, or even better the same variable we use to locate the runcards).

https://github.com/qiboteam/qibolab/blob/3416a8e3f6089104e2ccdf441b4f314a6d2f0d2c/src/qibolab/__init__.py#L21

However, for dummy we should mock the serialization, even when deployed: the installation folder might require privileges that the user does not have.
Furthermore, I start considering the option to split dummy from the regular Qibolab (we could not implement just in `tests/` because it's being used also by Qibocal... I'm thinking...)